docs/migration.md

```diff
@@ -458,4 +458,13 @@ ACPUserProfile.updateUserAttribute(attributeName: string, attributeValue: string
 - [AEP](https://github.com/adobe/aepsdk-react-native/tree/main/packages/userprofile/README.md#updateuserattributes)
 ```typescript
 UserProfile.updateUserAttributes(attributeMap: Record<string, any>)
-```
\ No newline at end of file
+```
+
+## Send data to Analytics
+There is no direct Analytics extension supported in Adobe Experience Platform SDK AEP-prefix React Native libraries. In order to send data to Analytics, there are two options we can achieve that.
+
+1. For new implementations of the Adobe Experience Platform SDK, it's highly recommended to send event data that is already XDM formatted using the `Edge.sendEvent` API to Edge Network Extension while set datastream directing to Adobe Analytics Service.
+Learn more the implementation at the [React Native Adobe Experience Platform Edge Network extension](../packages/edge).
+
+2. For migrating existing ACP-prefix react native libraries `MobileCore.trackAction` and `MobileCore.trackState` to AEP-prefix react native libraries, we can use Edge Bridge library which enables forwarding of Analytics track events through Experience platform Edge Network while set datastream directing to Adobe Analytics Service.
```

Comment:
You already covered these details in the Edge Bridge docs, we can skip this section here with the links I suggested above.